rmc/utils/mpc.py

```diff
@@ -363,11 +238,10 @@ def prepare_pop_path_ht(
     logger.info("Getting transcript annotations...")
     # If OE-annotated VEP context HT filtered to missense variants
     # in canonical transcripts does not exist, create it
-    if not file_exists(context_with_oe_dedup.path):
-        create_context_with_oe(
-            overwrite_temp=overwrite_temp,
-            overwrite_output=overwrite_output,
-        )
+    if not file_exists(context_with_oe_dedup.path) or not file_exists(
```

Comment:
we should add to the docstring that this function now assumes these resources exist. also, should we add freeze here (and to the function call [here](https://github.com/broadinstitute/regional_missense_constraint/blob/main/rmc/pipeline/calculate_mpc.py#L39))?

Reply:
Added changes to address both